### PR TITLE
[plugin.video.101barz] 1.0.0

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
1.0.0 (2019-02-13)
- initial version
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.101barz/default.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        for paramsPair in param_pairs:
+            param_splits = paramsPair.split('=')
+            if (len(param_splits)) == 2:
+                param_dict[param_splits[0]] = param_splits[1]
+    return param_dict
+
+
+def add_link(title, url, mode, thumbnail_image_url, desc="", duration=""):
+    u = sys.argv[0] + "?url=" + urllib.parse.quote_plus(url) + "&mode=" + convertToUnicodeString(mode)
+    liz = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image_url)
```
Comment (enen92) [inline plugin.video.101barz/default.py]: thumbnailImage is also deprecated (not only iconImage)

File: plugin.video.101barz/default.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    title = title.encode('ascii', 'ignore')
+    # Add refresh option to context menu
+    liz.addContextMenuItems([('Refresh', 'Container.Refresh')])
+    ok = xbmcplugin.addDirectoryItem(handle=int(sys.argv[1]), url=u, listitem=liz)
+    return ok
+
+
+def add_dir(title, url, mode, thumbnail_image_url, desc=""):
+    u = sys.argv[0] + '?url=' + urllib.parse.quote_plus(url) + '&mode=' + convertToUnicodeString(mode)
+    liz = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image_url)
```
Comment (enen92) [inline plugin.video.101barz/default.py]: same

File: plugin.video.101barz/default.py (hunk 1 above)
Comment (skipmodea1) [reply plugin.video.101barz/default.py]: Fixed.

File: plugin.video.101barz/default.py (hunk 2 above)
Comment (skipmodea1) [reply plugin.video.101barz/default.py]: Fixed.

**Pull request merged**